### DISCUSSION TITLE
dump modal doesn't work using livewire ESM build

### DISCUSSION THREAD

**Discussion by jdiddydave** [livewire/livewire#7459] [7. Bugs (archived)]
### Livewire version

3.x

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

When bundling LW in app.js using ESM build, putting `dump()` in an updated hook throws a browser console error.

Paired with @joshhanley on this.

![Screen Shot 2023-12-05 at 7 10 50 PM](https://github.com/livewire/livewire/assets/98569588/42e62af8-0513-44a9-b2f1-b8c28176691d)

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by luanfreitasdev** [comment]
https://github.com/livewire/livewire/pull/7626
